docs/Tutorial_Time_Series_Snippets.ipynb

```diff
@@ -1,5 +1,29 @@
 {
```

Comment:
<p>@ninimama I think this is pretty good and here are a couple of comments:</p><p> </p><ul><li>In your opening sentence, it is claimed that it is a "common question" but it would be nice to provide some (obvious) real-world examples if possible</li><li>I like the point that you are trying to make about snippets vs clustering but the clustering point could be a bit clearer</li><li>"Although a clustering" should be "Although clustering"</li><li>Since we acknowledge the original papers in the first sentence, there is no need to repeat things like "Authors in Matrix Profile XIII". We make no claims of these ideas being our own original ideas </li><li>So, I don't think the final tutorial will go through the creation of the toy data. We will just create it in Zenodo.ipynb and then download the final data to analyze</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/397/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
@seanlaw 

> * So, I don't think the final tutorial will go through the creation of the toy data. We will just create it in Zenodo.ipynb and then download the final data to analyze

In my recent commit, I keep the toy data for now and just add textual context and improve the current flow/figures. I tried to follow the same structure used in the other tutorials. 

I can take care of retrieving the data from Zenodo in the future commits (If you have any suggestion regarding this matter, please let me know) 

================================================

I have one question that has been bothering me:

Sometimes I try to limit the changes into one particular area to avoid the term "and" in the commit message. In other words, I would like to consider each commit for one particular issue. So, it would be easier for the reviewer (you) to go through the changes in the commit. However, in some occasions, there are only a few changes that need to be done for a particular issue. For instance, adding the heading to the notebook is probably less than 10 lines. Or, modifying figures (and change their orders) is again less than 10 lines. This can easily happen in different scenarios, when I try to improve different parts of the Tutorial. 

Is that okay to do one git commit for all those small changes? (This is what I did in my recent commit that I pushed) 

OR: Can I do three small commits (each corresponds to small changes). Then, after a few commits, I can do git push. Since I don't know how the review process looks like on your end, I have no idea if the latter approach makes sense or not.